src/ensemblGenomeBrowser.ts

```diff
@@ -0,0 +1,53 @@
+import { 
+    ConfigData,
+    GenomeBrowserType,
+    IncomingActionType,
+    OutgoingAction,
+    Subscribe,
+    SubscribeArgs
+  } from './types';
+  
+  import send from './methods/send';
```

Comment:
Starting from this import and down, everything is indented two spaces to the right.